src/routes/Register/container.js

```diff
@@ -30,6 +30,7 @@ export class RegisterContainer extends Component {
 		await dispatch({ loading: true, department });
 		try {
 			await Livechat.grantVisitor({ visitor: { ...fields, token } });
+			await Livechat.updateVisitorSessionOnRegister({ visitor: { ...fields, token } });
```

Comment:
Why not just one method to update the livechat session? 
You just need one method, you can pass specific data as parameters.